lib/cc/engine/flog.rb

```diff
@@ -90,10 +89,11 @@ def run
         Dir.chdir dir do
           flog.flog(*files)
 
-          flog.each_by_score flog.threshold do |name, score, call_list|
+          flog.each_by_score do |name, score, call_list|
             location = flog.method_locations[name]
 
             next unless location # XXX#main is location-less, skip for now
+            next unless score > config["max_score"]
```

Comment:
heh yea it all depends on perspective. it's either the "maximum allowed complexity" (how I intended it) or the "minimum problematic complexity"... agreed it's ambiguous and should be changed. I'd love to just use "threshold" except some people will already have values committed like "0.6" which isn't appropriate for this kind of threshold. Going to tentatively change this to "score_threshold". WDYT?